lib/linked-list/list.rb

```diff
@@ -136,6 +160,11 @@ def each
       __each { |node| yield(node.data) }
     end
 
+    def each_node
```

Comment:
Could you please add short documentation for this method? Similar to `each`.

Reply:
done 👍

Reply:
Thanks! I'll take a closer look tonight and then release a new version.